pkg/pillar/containerd/oci.go

```diff
@@ -144,10 +144,12 @@ func (s *ociSpec) AddLoader(volume string) error {
 		for _, e := range s.Process.Env {
 			keyAndValueSlice := strings.SplitN(e, "=", 2)
 			if len(keyAndValueSlice) == 2 {
-				//handles Key=Value case
-				envContent = envContent + fmt.Sprintf("export %s=\"%s\"\n", keyAndValueSlice[0], keyAndValueSlice[1])
+				// handles Key=Value case
+				// We should not quote the value twice, so let's trim off first
```

Comment:
For those like me which have to look up what Trim does it might make sense to add that we remove any leading and trailing double quotes (but leave internal quotes untouched).
I wounder if we should remove leading and trailing space as well in case a user does
CMD= "a b"
??

Reply:
good points @eriknordmark, I will test with leading spaces, but I think it's a good idea to also trim off them, it can be done in the same function call.... I will come back with the results here, we can't wait to merge this PR....

Reply:
nice catch @eriknordmark ! I've tested and `CMD= "a b"` will not trim off the double quotes and it will fail... so we should trim off white spaces as well... I will update the PR with the fix....